src/Components/Article/Article.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useState } from "react";
+import Header from "../Header";
+import { useParams } from "react-router-dom";
+import axios from "axios";
+
+export default function Article() {
+  let { article_id } = useParams();
+  const [currArticle, setCurrArticle] = useState({});
```

Comment:
What would you expect to see while you were waiting for the article to be fetched?